mnefun/_utils.py

```diff
@@ -80,6 +80,19 @@ def _handle_dict(entry, subj):
     return out
 
 
+def _handle_decim(decim, sfreq):
+    decim = np.array(decim)
+    assert decim.shape == ()
+    if decim.dtype.char in 'il':
```

Comment:
Again, for my own personal edification: what is `il`?

Reply:
int or long

Reply:
Maybe comment w/ _NumPy dtype attribute for workspace names_ if that's the correct terminology. That's some Kung Fu Panda shit :)